results/pkg/api/server/v1alpha1/server.go

```diff
@@ -401,6 +419,18 @@ func (s Server) getResultByID(name string) (*pb.Result, error) {
 	return result, nil
 }
 
+func getResultETag(result *pb.Result) (etag string, err error) {
+	tmpEtag := result.GetEtag()
+	result.Etag = ""
+	if b, err := proto.Marshal(result); err == nil {
+		dst := make([]byte, base64.RawURLEncoding.EncodedLen(len(b)))
+		base64.RawURLEncoding.Encode(dst, b)
+		etag = base64.RawURLEncoding.EncodeToString(dst)
```

Comment:
I'm slightly worried that this will inflate the size of a record, since we're effectively duplicating the data.

This is probably fine for a first pass, but we should look into a slimmed down identifier in the future (perhaps update timestamp?)